wordpress-mobile/WordPress-Android
# Inline action of Like Post

Part of #20024 

This PR contains:
1. Implement the inline action of Like Post
2. Refactors (mostly in NoteAdapter)
-----

## To Test:

1. Open the Jetpack app and switch to the notifications tab.
2. Find a post and click on the star icon in the notification item.
3. The Like status should be updated.
4. Done, thank you!

-----

## Regression Notes

1. Potential unintended areas of impact

    - Notification tab

3. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Manual

4. What automated tests I added (or what prevented me from doing so)

    - None

-----

## PR Submission Checklist:


-----

## Testing Checklist:
